libexec/klab-compress

```diff
@@ -31,15 +31,14 @@ if [ ! -d "$KLAB_OUT/log" ]; then
     mkdir -p "$KLAB_OUT/log"
 fi
 
-zip -r "$KLAB_OUT/log/$spec_hash.zip" config.json
-
 cd $KLAB_OUT
 find . \
-     \( -wholename "*$spec_hash*" -or -wholename "*$spec_name*" \) \
-     -and ! -wholename "*/log/*.zip" \
-     -exec zip "./log/$spec_hash.zip" {} +
-
-zip -r "log/$spec_hash.zip" \
-    "prelude.smt2" \
-    "rules.k" \
-    "bin_runtime.k"
+  \( -name "*$spec_hash*" \
+  -o -name "*$spec_name*" \
+  -o -name config.tmp.json \
+  -o -name prelude.smt2 \
+  -o -name rules.k \
+  -o -name bin_runtime.k \) \
+  -a \! -name "*.tar" \
+  -a \! -name "*.zip" \
+  -exec tar --create --zstd --file "./log/$spec_hash.tar" {} +
```

Comment:
Here we could pass the `--threads=N` argument to zstd, to control how many threads are used for compression, in case we deemed it necessary.

Reply:
do you have a rough idea of what kind of speedup we could get from this?

Reply:
No, but I didn't notice 

> It seems like it would be fairly easy to add some conditional logic in `klab fetch` that can handle both?

Yes, we could do that if needed. Do we need to? @livnev opinions?

Reply:
On the test archive on buildbot, using `-T0`, 29 vs 33 seconds, so negligible. Maybe there are other bottlenecks.